bagisto/bagisto
# Add locale to Category datagrid

## Description
Link: #5069 

- Locale filter added to category datagrid
- Locale added to input labels in category edit page


## How to test this
Add a new locale to your channel then edit a category and change local, change some locale-based values and save, after getting back to datagrid you can filter locale and see your changes.
You also may need to do `php artisan vendor:publish --all --force`
